azure-pipelines.yml

```diff
@@ -1,8 +1,11 @@
 trigger:
-  - "*"
+  - master
```

Comment:
Isn't this the opposite of what we just did in foyer?

Reply:
I don't think we changed this in foyer. 

I reverted back to using `master` as the trigger based on our chat with @umesh-timalsina on slack, since whatever issues we had seemed to be resolved on his branch (https://github.com/umesh-timalsina/mbuild/pull/2)

Reply:
[This File](https://github.com/umesh-timalsina/mbuild/blob/631-azp-fix/azure-pipelines.yml) adds PR trigger against master (Opposed to *). Updates `conda` to the latest version available (With a CI Fix from pandas https://github.com/pandas-dev/pandas/commit/74bbbb084dc530080da93b96a41114d0598f928b). On my fork, its working for PR, Scheduled builds are not failing.